tidb-binlog/bidirectional-replication-between-tidb-clusters.md

```diff
@@ -9,9 +9,8 @@ aliases: ['/docs/dev/tidb-binlog/bidirectional-replication-between-tidb-clusters
 > **Warning:**
 >
 > - Currently, bidirectional replication is still an experimental feature. It is **NOT** recommended to use it in the production environment.
+> - Starting from TiDB v8.3.0, [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) is deprecated, and is planned to be removed in a future release. Use [TiCDC](/ticdc/ticdc-overview.md) instead for incremental data replication. Use [Point-in-Time Recovery (PITR)](/br/br-pitr-guide.md) for the PITR scenario needs.
```

Comment:
Replication was deprecated in v7.5.0, not v8.3.0, so this specific content about replication should reflect that